packages/react-storage/src/components/StorageBrowser/context/primitives.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  PrimitiveButton,
+  PrimitiveView,
+  createPrimitivesContext,
+  DefaultPrimitives,
+} from '@aws-amplify/ui-react/internal';
+
+export interface StorageBrowserPrimitives
+  extends Pick<DefaultPrimitives, 'Button' | 'View'> {}
+
+const defaultValue: StorageBrowserPrimitives = {
+  Button: PrimitiveButton,
+  View: PrimitiveView,
+};
```

Comment:
sorry if this was mentioned somewhere else, but what are the Button and View supposed to represent? This context isn't being used in this PR at the moment right?

Reply:
That's correct, this is just scaffolding/adding as examples